hydra-node/src/Hydra/Chain/Direct/State.hs

```diff
@@ -103,6 +103,7 @@ data HydraStateMachine (st :: HeadStateKind) where
     { initialThreadOutput :: InitialThreadOutput
     , initialInitials :: [UTxOWithScript]
     , initialCommits :: [UTxOWithScript]
+    , initialCommitted :: UTxO
```

Comment:
We don't need to store the committed UTxO anymore as we can reconstruct the order from the datums